routes/originNuget.js

```diff
@@ -4,6 +4,7 @@
 const asyncMiddleware = require('../middleware/asyncMiddleware')
 const router = require('express').Router()
 const requestPromise = require('request-promise-native')
+const utils = require('../lib/utils')
```

Comment:
Same here, it doesn't need to check the uniqueness of the results?

Reply:
Have included implementation here now also